sdk/lib/rtl/actctx.c

```diff
@@ -5394,38 +5435,40 @@ NTSTATUS WINAPI RtlZombifyActivationContext( HANDLE handle )
     return STATUS_NOT_IMPLEMENTED;
 }
 
-NTSTATUS
-NTAPI RtlActivateActivationContextEx( ULONG flags, PTEB tebAddress, HANDLE handle, PULONG_PTR cookie )
+/******************************************************************
+ *		RtlActivateActivationContext (NTDLL.@)
+ */
+NTSTATUS WINAPI RtlActivateActivationContext( ULONG unknown, HANDLE handle, PULONG_PTR cookie )
+{
+    return RtlActivateActivationContextEx( 0, NtCurrentTeb(), handle, cookie );
+}
```

Comment:
I know this would result in another winediff, but we had the first argument as 'flags' and passed it on to the `Ex` version, would that be worth doing again?